package/samplers/synetune_sampler/requirements.txt

```diff
@@ -0,0 +1,3 @@
+optuna
+optunahub
+syne-tune==0.13.0
```

Comment:
Just to be sure, does this works with all the searchers you have? Or do you need a tag? (if so we should add it)

Reply:
Maybe `syne-tune>=0.13.0` instead would be more flexible?